schema.sql

```diff
@@ -10,3 +10,28 @@ CREATE TABLE animals(
     );
     
     ALTER TABLE animals ADD species VARCHAR(50);
+
+
+    /* create table for owners' data.*/
+CREATE TABLE owners( 
+    id                  INT GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1), 
+    full_name            VARCHAR(50), 
+    age                  INT,
+    );
+
+    /* create table for species' data.*/
+CREATE TABLE species( 
+    id                  INT GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1), 
+    name                 VARCHAR(50), 
+    );
+
+
+ALTER TABLE animals ADD PRIMARY KEY (id);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD species_id INT;
+ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id);
+
+ALTER TABLE animals ADD owner_id INT;
+ALTER TABLE animals ADD CONSTRAINT fk_owner FOREIGN KEY(owner_id) REFERENCES owners(id);
```

Comment:
Foreign key added successfully referencing proper tables 📝

Reply:
I'm not sure if I am suppose to change anything here. Could you please confirm or otherwise? Thanks